src/main/resources/assets/pmmo/lang/pt_br.json

```diff
@@ -5,18 +5,18 @@
     "pmmo.levelUpDamageBoostPercentMagic": "%1$s %2$s upou o n\u00edvel! %3$s%% B\u00f4nus em magia!",
     "pmmo.notSkilledEnoughToUseTinkersMaterial": "Voc\u00ea n\u00e3o \u00e9 habilidoso o bastante para usar um %1$s material do tinkers !",
     "pmmo.halfHeartRegenerationSeconds": "%1$s Regenera\u00e7\u00e3o de HP por 60s",
-    "pmmo.welcomeText": "Welcome! %1$s to learn more about Project MMO!",
-    "pmmo.clickMe": "Click Me",
-    "pmmo.notSkilledEnoughToInteractWith": "You are not skilled enough to interact with %1$s!",
-    "pmmo.info": "Extra Info",
-    "pmmo.openInfo": "Open GUI Info Screen",
-    "pmmo.patreon": "Become a Patreon!",
-    "pmmo.donate": "Support My Work!",
-    "pmmo.discord": "Join Our Discord!",
-    "pmmo.features": "Project MMO Features List!",
-    "pmmo.customize": "Customize Your Experience!",
-    "pmmo.curseforge": "Project MMO CurseForge Page!",
-    "pmmo.outdatedVersion": "Project MMO version is outdated! Latest: %1$s, Installed: %2$s!",
+    "pmmo.welcomeText": "Bem-vindo! %1$s para saber mais sobre o Projeto MMO!",
+    "pmmo.clickMe": "Clique em mim",
+    "pmmo.notSkilledEnoughToInteractWith": "Você não é habilidoso o suficiente para interagir com %1$s!",
+    "pmmo.info": "Informação extra",
+    "pmmo.openInfo": "Abrir tela de informações da GUI",
```

Comment:
This should be just "Abrir tela de informações"

Reply:
@FITFC please see the above and correct.